engine-rest/engine-rest-openapi/src/main/templates/paths/history/job-log/{id}/stacktrace/get.ftl

```diff
@@ -0,0 +1,56 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/job-log/get-job-log-stacktrace/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getStacktraceHistoricJobLog"
+      tag = "Historic Job Log"
+      summary = "Get Job Log Exception Stacktrace"
+      desc = "Retrieves the corresponding exception stacktrace to the passed historic job log by
+              id."
+  />
+
+  "parameters" : [
+
+      <@lib.parameter
+          name = "id"
+          location = "path"
+          type = "string"
+          required = true
+          desc = "The id of the historic job log to get the exception stacktrace for."
+          last = true
+      />
+
+  ],
+
+  "responses": {
+
+    <@lib.response
+        code = "200"
+        mediaType= "text/plain"
+        desc = "Request successful."
+        examples = ['"example-1": {
+                       "summary": "response",
+                       "description": "GET `history/job-log/someId/stacktrace`
+
+                                       The result is the corresponding stacktrace as plain text.",
+                       "value": "java.lang.RuntimeException: A exception message!
+                                  at org.camunda.bpm.pa.service.FailingDelegate.execute(FailingDelegate.java:10)
+                                  at org.camunda.bpm.engine.impl.delegate.JavaDelegateInvocation.invoke(JavaDelegateInvocation.java:34)
+                                  at org.camunda.bpm.engine.impl.delegate.DelegateInvocation.proceed(DelegateInvocation.java:37)
+                                  ..."
+                     }']
+    />
+
+    <@lib.response
+        code = "404"
+        dto = "ExceptionDto"
```

Comment:
This creates a return type `application/json`, right? The docs state (and the Java code implies) that we return plain text instead. I have not given it a try with the API itself to validate that behavior. But it might be wrong then here. Maybe this is something that could be double-checked.

Reply:
Good catch! I believe the docs are incorrect. The Java Code throws an invalid InvalidRequestException object, which turns into application/json if I am not mistaken?

Reply:
That's what I thought as well but then again the method is annotated to return plain text, so, no clue what wins. Maybe you can give that an actual try? 😃

Reply:
Yup. I checked and it is application/json

Reply:
Awesome! 👍 🚀